test/sass/_sections.scss

```diff
@@ -13,15 +13,15 @@
 
       @include expect {
         // sass-lint:disable-all
-        border-bottom: 0.44444rem solid #e07;
+        border-bottom: size('border') solid #e07;
```

Comment:
I didn't know you could use uncompiled sass stuff here. Thought you had to use the value you expect in these test files?